sat/logging.py

```diff
@@ -26,6 +26,8 @@
 """
 import logging
 import os
+from copy import deepcopy
+from logging import Formatter, Handler, LogRecord
```

Comment:
Unused `Handler` import.

```suggestion
from logging import Formatter, LogRecord
```